urllib3/exceptions.py

```diff
@@ -193,6 +193,14 @@ class ResponseNotChunked(ProtocolError, ValueError):
     pass
 
 
+class IncompleteRead(HTTPError):
```

Comment:
This should probably inherit from `httplib`'s `IncompleteRead` error class, which we already sometimes raise.